src/main/java/uk/gov/justice/hmpps/casenotes/repository/OffenderCaseNoteRepository.kt

```diff
@@ -14,12 +14,24 @@ import java.util.*
 
 @Repository
 interface OffenderCaseNoteRepository : PagingAndSortingRepository<OffenderCaseNote, UUID>, JpaSpecificationExecutor<OffenderCaseNote> {
+
   fun findBySensitiveCaseNoteType_ParentType_TypeInAndModifyDateTimeAfterOrderByModifyDateTime(types: Set<String>?, createdDate: LocalDateTime?, page: Pageable?): List<OffenderCaseNote>
+
   fun findByModifyDateTimeBetweenOrderByModifyDateTime(fromDateTime: LocalDateTime, toDateTime: LocalDateTime): List<OffenderCaseNote>
 
+  @Query(value = "SELECT ocn.* FROM OFFENDER_CASE_NOTE ocn WHERE ocn.offender_case_note_id = ?1", nativeQuery = true)
+  fun findByIdIgnoreSoftDelete(offenderCaseNoteId: UUID): Optional<OffenderCaseNote>
+
   @Modifying
   @Query("UPDATE OffenderCaseNote ocn SET ocn.offenderIdentifier = :newOffenderIdentifier WHERE ocn.offenderIdentifier = :oldOffenderIdentifier")
```

Comment:
wondering if we need to change this to native sql so works for soft deleted records too?